src/components/vanilla/controls/DatePicker/components/DateRangePicker.tsx

```diff
@@ -37,7 +37,7 @@ type Props = {
   hideDate?: boolean;
 };
 
-export default (props: Props) => {
+export default function DateRangePicker(props: Props) {
```

Comment:
<img width="1103" alt="image" src="https://github.com/user-attachments/assets/456392a4-dda7-4648-9871-63c6b6cdbd49">

run the cmd "npm run lint", there are some linting issues

Reply:
@mad-raccoon I fixed this so not sure why it doesn't show as resolved here

Reply:
no idea, but you did well resolving it manually !